public/app/core/components/Page/types.ts

```diff
@@ -25,6 +25,8 @@ export interface PageProps extends HTMLAttributes<HTMLDivElement> {
   layout?: PageLayoutType;
   /** Can be used to get the scroll container element to access scroll position */
   onSetScrollRef?: (ref: ScrollRefElement) => void;
+  /** Set a page-level toolbar */
+  toolbar?: React.ReactNode;
```

Comment:
I wondered if this was actually exposing `toolbar` prop to plugins (which will be fine eventually, but not yet), but Plugins actually use [PluginPage](https://github.com/grafana/grafana/blob/47f87171496f675bc44ddf839effaed1846bfa4e/public/app/core/components/Page/PluginPage.tsx) which is constrained to pass through only limited props 👍